pest/src/iterators/pair.rs

```diff
@@ -135,7 +135,7 @@ impl<'i, R: RuleType> Pair<'i, R> {
     /// #     state.rule(Rule::ab, |s| s.match_string("ab"))
     /// }).unwrap().next().unwrap();
     ///
-    /// assert_eq!(pair.into_span().as_str(), "ab");
+    /// assert_eq!(pair.as_span().as_str(), "ab");
```

Comment:
This got caught up in my refactoring (the doc-test for `Pair::into_span`). I'm unsure which we should use here.

Reply:
I'd leave the old one here.